pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -251,6 +238,42 @@ async def run(self):
             dry_run=self.dry_run,
         )
 
+    def regen_repo_with_complete(self, el_v, timeout):
```

Comment:
we can make this async, so we can fire this off for both el versions in parallel